packages/@react-spectrum/card/src/CardView.tsx

```diff
@@ -195,6 +196,20 @@ function InternalCard(props) {
     focusMode: 'cell'
   }, state, unwrappedRef);
 
+  // Prevent space key from scrolling the CardView if triggered on a disabled item or on a Card in a selectionMode="none" CardView.
+  let allowsInteraction = state.selectionManager.selectionMode !== 'none';
+  let isDisabled = !allowsInteraction || state.disabledKeys.has(item.key);
+
+  let onKeyDown = (e) => {
+    if (e.key === ' ' && isDisabled) {
+      e.preventDefault();
+    }
+  };
```

Comment:
Could do a `usePress` call here instead similar to what TableRow does but felt unnecessary since we don't have a active state for the Card

Reply:
Could this be in react-aria? This can't be moved to CardView can it? I'm asking if we care about preventing space from scrolling when a card isn't in a CardView. What prompted me to ask this is I was thinking we should already know if a card is "disabled", but within a CardView we'd have to ask the card (really CardView state which stores this info for the card.

Reply:
This could perhaps be moved into `useSelectableItem` to cover general selectable items and into `useCard` for the standalone card but I'll defer that until we feel like this is appropriate behavior for all selectable items and when we finalize standalone cards